profile.md

```diff
@@ -111,23 +111,23 @@ One item not addressed in detail in this document is how the issuer decides on w
   <tr>
    <td><strong>Issuer </strong>
    </td>
-   <td>Any token issuer, this refers to both Authorization Servers and OpenID Providers.
+   <td>Any token issuer, refers to both Authorization Servers and OpenID Providers.
    </td>
-   <td>VO https url
+   <td>VO https URL
    </td>
   </tr>
   <tr>
    <td><strong>Authorization Server</strong>
    </td>
-   <td>The entity which produces (“issues”) the token. For WLCG authorization, this is a service run by the VO that is asserting the identity or the authorization to access the VO’s resources. This term is defined by OAuth2 and may be referred to as the Authorization Server.
+   <td>The entity that produces (“issues”) the token. For WLCG authorization, this is a service run by the VO that is asserting the identity or the authorization to access the VO’s resources. This term is defined by OAuth2, sometimes referred also as Authorization Provider. Is equivalent to an issuer in OIDC terminology.
```

Comment:
"referred also as" doesn't make for a good sentence.  I suggest "referred to as".

Reply:
done